### PR TITLE
Mw/issue175

### PR DESCRIPTION
**Pull request by maxence-wz**

### CODE REVIEW

File: mtest/src/PipeTest.cxx [hunk 1]
```diff
@@ -1580,13 +1608,31 @@ namespace mtest {
...
+      if ((!this->hpp) && (c == Configuration::CURRENT_CONFIGURATION)) {
+        return state.u1[0];
+      }
+      return this->mesh.inner_radius;
+    }();
+    const auto re = [this, state, c]() -> real {
+      if ((!this->hpp) && (c == Configuration::CURRENT_CONFIGURATION)) {
+        const size_type ln = this->getNumberOfNodes() - 1;
+        const auto Re = this->mesh.outer_radius;
+        return (this->hpp) ? Re : Re + state.u1[ln];
```
Comment (thelfer) [inline mtest/src/PipeTest.cxx:1624]: shall be return Re+....
(testing this->hpp is meaningless here)

File: mtest/src/PipeTest.cxx [hunk 2]
```diff
@@ -1580,13 +1608,31 @@ namespace mtest {
...
-                                  const std::string& n) const {
+                                  const std::string& n,
+                                  const Configuration c) const {
     constexpr real pi = 3.14159265358979323846;
-    const auto ri = this->mesh.inner_radius;
-    const auto re = this->mesh.outer_radius;
-    const auto h = 1;
+    const auto ri = [this, state, c]() -> real {
+      if ((!this->hpp) && (c == Configuration::CURRENT_CONFIGURATION)) {
+        return state.u1[0];
```
Comment (thelfer) [inline mtest/src/PipeTest.cxx:1616]: This is wrong. Shall be mesh.inner_radius+state.u1[0]

**Comment by thelfer** [comment]
@maxence-wz Would you check my two comments ?

File: mtest/src/PipeTest.cxx (hunk 1 above)
Comment (maxence-wz) [reply mtest/src/PipeTest.cxx]: see the commit https://github.com/thelfer/tfel/pull/252/commits/407a9bc65dca6aca5d901b219889fc36e5599ba4

File: mtest/src/PipeTest.cxx (hunk 2 above)
Comment (maxence-wz) [reply mtest/src/PipeTest.cxx]: see the commit https://github.com/thelfer/tfel/pull/252/commits/407a9bc65dca6aca5d901b219889fc36e5599ba4

**Comment by maxence-wz** [comment]
@thelfer I solved the conflicts and updated `docs/mtest/ptest/AdditionalOutputs.md`

**Comment by thelfer** [comment]
@maxence-wz Do not use local paths in your tests. Please check this and launch the test suite in your install directory. Once done, merge seems ok.

**Pull request merged**